src/columnChart/baseColumnChart.ts

```diff
@@ -1437,7 +1437,7 @@ export class BaseColumnChart implements IColumnChart {
         return this.data;
     }
 
-    private checkDataToFeatures(): boolean {
+    public checkDataToFeatures(): boolean {
```

Comment:
why?

Reply:
I am using this function to only allow setting category gradients if there is only one of each series in a category